includes/wikia/tests/UserPasswordTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\DependencyInjection\Injector;
+
+class UserPasswordTest extends WikiaBaseTest
+{
+
+	const TEST_USER_ID = 2257663;
+
+	/** @var User */
+	protected $testUser;
+
+	protected static $currentInjector;
+
+	public static function setUpBeforeClass() {
+		self::$currentInjector = Injector::getInjector();
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->testUser = User::newFromId( self::TEST_USER_ID );
+	}
+
+	public static function tearDownAfterClass() {
+		Injector::setInjector( self::$currentInjector );
+	}
+
+	public function testShouldSetNewPassword() {
+		$this->assertTrue( $this->testUser->setPassword( "ok" ) );
```

Comment:
Also, throughout this code, single quotes should be preferred when not using string interpolation per coding conventions.